src/pyclaw/classic/solver.py

```diff
@@ -1,8 +1,8 @@
 r"""
```

Comment:
I am actually in general not against doing refactoring of white space but in this case it obfuscates the actual fix you did.  If you want to do a white space removal pass I would put in a different PR and only make the change for the `aux` array here.

Reply:
what is a "white space removal pass"?